### ISSUE TITLE
feature request: enable/disable BPM counter

### ISSUE THREAD

**Issue by deejaygirl** [mixxxdj/mixxx#12078]
### Feature Description

Hi

For people trying to learn beatmatching, it could be useful to have a setting that enables/disables (enabled by default) the BPM counter.
In this way there is no visual clue when beatmatching just by hearing.

This is my small feature request.
If the BPM counter can't be removed, it can maybe display "---" when disabled so the UI doesn't have to change.

**Comment by ronso0** [comment]
I used to stick PostIts onto the BPM displays back then X)

Then I hacked the LateNight skin.
Nowadays you can hide the rate section entirely in LateNight and Tango skin.
 IIRC Deere had a 'obscure BPM' option. All these options can be found in the skin settings menu which be toggled with the cog icon in the top right corner, or with Ctrl+1

**Comment by deejaygirl** [comment]
> All these options can be found in the skin settings menu which be toggled with the cog icon in the top right corner, or with Ctrl+1

I have no idea what you mean. I don't find the skin settings menu

**Comment by deejaygirl** [comment]
So an option to disable it , wouldn't that be much easier?

**Comment by ronso0** [comment]
you found it
"Settings" in the skin = skin settings
Just uncheck Rate Controls.

**Comment by deejaygirl** [comment]
> Just uncheck Rate Controls.

Thanks for your help but doing this also REMOVES the pitch slider :)
How you suggest we learn beatmatching without any slider? :-D

**Comment by ronso0** [comment]
Oh, you're practing beat matching with the GUI slider... somehow I assumed you got your DDJ-200 and use that.

So this is a valid request IMO. Though I doubt anyone will implement it in LateNight, maybe someone will for the new QML interface.

For now you can use the Deere skin: uncheck **BPM** and the BPM display is obscured
![image](https://github.com/mixxxdj/mixxx/assets/5934199/0fc94aa3-d0fa-4702-b884-39a9e419610f)

If you desperately need it in LateNight you need to modify it. If you want to go that route I can guide you, but that should happen in a forum thread, not here.

**Comment by deejaygirl** [comment]
> Oh, you're practing beat matching with the GUI slider... somehow I assumed you got your DDJ-200 and use that.
> 
> So this is a valid request IMO. Though I doubt anyone will implement it in LateNight, maybe someone will for the new QML interface.

Yes, this feature request is for people wanting to practice beatmatching using Mixxx with no controller!
You must have a lot of luck to find this is possible in Deere skin (you don't expect it's possible in only 1 skin...)

> 
> For now you can use the Deere skin: uncheck **BPM** and the BPM display is obscured ![image](https://user-images.githubusercontent.com/5934199/273476666-0fc94aa3-d0fa-4702-b884-39a9e419610f.png)

Thanks! This is what I wanted.

> 
> If you desperately need it in LateNight you need to modify it. If you want to go that route I can guide you, but that should happen in a forum thread, not here.

Why does each skin have its own settings? Are the skins not made the same but other colors and widgets?
You don't want to add the "BPM" setting in LateNight like in Deere, since you know how to do it? The LateNight skin is the most beautiful one.
I have no interest in doing it myself, I'm not a developer and don't know anything about Mixxx internal stuff.
